pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.ruff]
+# Only lint notebooks (and optionally your Python code)
+include = ["notebooks/**/*.ipynb"]
+line-length = 100

```

Comment:
120 would be more typical. Would that let you remove the ignore for E501?

Reply:
yes. Thanks. never thought of that.